notebooks/02_model/rbm_deep_dive.ipynb

```diff
@@ -0,0 +1,1160 @@
+{
```

Comment:
Small comment here, can you move all the figures to the common blob that we have? https://recodatasets.blob.core.windows.net/images the credentials are in the wiki on teams